packages/contracts/src/plugins/governance/majority-voting/token/build-metadata.json

```diff
@@ -1,13 +1,105 @@
+            },
+            {
+              "internalType": "uint64",
+              "name": "minDuration",
+              "type": "uint64",
+              "description": "The minimum duration of the proposal vote in seconds."
+            },
+            {
+              "internalType": "uint256",
+              "name": "minProposerVotingPower",
+              "type": "uint256",
+              "description": "The minimum voting power required to create a proposal."
+            }
+          ],
+          "internalType": "struct MajorityVotingBase.VotingSettings",
+          "name": "_votingSettings",
+          "type": "tuple",
+          "description": "The majority voting settings that will be applied as parameters on proposal creation."
+        },
+        {
+          "components": [
+            {
+              "internalType": "address",
+              "name": "token",
+              "type": "address",
+              "description": "The token address. If this is `address(0)`, a new `GovernanceERC20` token is deployed. If not, the existing token is wrapped as an `GovernanceWrappedERC20`."
+            },
+            {
+              "internalType": "string",
+              "name": "name",
+              "type": "string",
+              "description": "The token name. This parameter is only relevant if the token address is `address(0)`."
+            },
+            {
+              "internalType": "string",
+              "name": "symbol",
+              "type": "string",
+              "description": "The token symbol. This parameter is only relevant if the token address is `address(0)`."
+            }
+          ],
+          "internalType": "struct TokenVotingSetup.TokenSettings",
+          "name": "tokenSettings",
+          "type": "tuple",
+          "description": "The token settings that either specify an existing ERC-20 token (`token = address(0)`) or the name and symbol of a new `GovernanceERC20` token to be created."
+        },
+        {
+          "components": [
+            {
+              "internalType": "address[]",
+              "name": "receivers",
+              "type": "address[]",
+              "description": "The receivers of the tokens."
+            },
+            {
+              "internalType": "uint256[]",
+              "name": "amounts",
+              "type": "uint256[]",
+              "description": "The amounts of tokens to be minted for each receiver."
+            }
+          ],
+          "internalType": "struct GovernanceERC20.MintSettings",
+          "name": "_mintSettings",
+          "type": "tuple",
+          "description": "The token mint settings struct containing the `receivers` and `amounts`."
+        }
+      ],
+      "prepareUpdate": {
+        "1": {
+          "description": "The information required for the update from build 1 to build 2.",
```

Comment:
"No new parameters are needed for this update" 

Or simply remove this field in general

Reply:
Fixed in aab50afe050c0a9a3a79b781e0aab86d0b198ed1